src/commands/moderation/prefixCommands/functions/modifyVersion.ts

```diff
@@ -0,0 +1,166 @@
+    title: 'Prefix Commands - Modify Version - Wrong format',
+    description: `The name and alias of a version can only contain alphanumerical characters, underscores and dashes. "${invalidString}" is invalid.`,
+    color: Colors.Red,
+});
+
+const doesNotExistsEmbed = (version: string) => makeEmbed({
+    title: 'Prefix Commands - Modify Version - Does not exist',
+    description: `The prefix command version ${version} does not exists. Can not modify it.`,
+    color: Colors.Red,
+});
+
+const alreadyExistsEmbed = (version: string, reason: string) => makeEmbed({
+    title: 'Prefix Commands - Add Version - Already exists',
+    description: `The prefix command version ${version} already exists: ${reason}`,
+    color: Colors.Red,
+});
+
+const successEmbed = (version: string, versionId: string) => makeEmbed({
+    title: `Prefix command version ${version} (${versionId}) was modified successfully.`,
+    color: Colors.Green,
+});
+
+const modLogEmbed = (moderator: User, version: string, emoji: string, alias: string, enabled: boolean, versionId: string) => makeEmbed({
+    title: 'Prefix command version modified',
+    fields: [
+        {
+            name: 'Version',
+            value: version,
+        },
+        {
+            name: 'Moderator',
+            value: `${moderator}`,
+        },
+        {
+            name: 'Emoji',
+            value: emoji,
+        },
+        {
+            name: 'Alias',
+            value: alias,
+        },
+        {
+            name: 'Enabled',
+            value: enabled ? 'Yes' : 'No',
+        },
+    ],
+    footer: { text: `Version ID: ${versionId}` },
+    color: Colors.Green,
+});
+
+const noModLogs = makeEmbed({
+    title: 'Prefix Commands - Modified Version - No Mod Log',
+    description: 'I can\'t find the mod logs channel. Please check the channel still exists.',
+    color: Colors.Red,
+});
+
+export async function handleModifyPrefixCommandVersion(interaction: ChatInputCommandInteraction<'cached'>) {
+    await interaction.deferReply({ ephemeral: true });
+
+    const conn = getConn();
+    if (!conn) {
+        await interaction.followUp({ embeds: [noConnEmbed], ephemeral: true });
+        return;
+    }
+
+    const version = interaction.options.getString('version')!;
+    const name = interaction.options.getString('name') || '';
+    const emoji = interaction.options.getString('emoji') || '';
+    const alias = interaction.options.getString('alias')?.toLowerCase() || '';
+    const enabled = interaction.options.getBoolean('is_enabled');
+    const moderator = interaction.user;
+
+    const nameRegex = /^[\w-]+$/;
+    if (name && !nameRegex.test(name)) {
+        await interaction.followUp({ embeds: [wrongFormatEmbed(name)], ephemeral: true });
+        return;
+    }
+    if (alias && !nameRegex.test(alias)) {
+        // eslint-disable-next-line no-await-in-loop
+        await interaction.followUp({ embeds: [wrongFormatEmbed(alias)], ephemeral: true });
```

Comment:
```suggestion
    if (alias && !nameRegex.test(alias)) {
        await interaction.followUp({ embeds: [wrongFormatEmbed(alias)], ephemeral: true });
```
I don't believe that ESLint directive is needed here?